tile_generator/package_flags.py

```diff
@@ -503,14 +505,15 @@ def _apply(self, config_obj, package, release):
                             'token': '(( .properties.k8s_cluster_token.value ))',
                             'service_id': '(( .properties.service_id.value ))',
                             'helm_chart_dir': '/var/vcap/packages/charts_for_%s/chart' % packagename,
+                            'operator_dir': '/var/vcap/packages/charts_for_%s/operator_chart' % packagename,
```

Comment:
You seem to be repurposing/piggybacking the charts_for_%s directory. Did you ensure there is no adverse consequence of that? Does it make sense to have a operator_chart_for_%s?

Reply:
The benefit for doubling-up in the charts_for_%s directory is that it allows for both directories to be handled in the loader errand.

After reviewing the code, there's a few changes that were missed, which only affected private repositories.  It's an additional parameter being passed to the loader errand definition and a change in the kibosh repository.